apps/super-app/src/common/components/CallHome.tsx

```diff
@@ -0,0 +1,5 @@
+const CallHome = () => {
+  return <h1>Call Home Page placeholder</h1>;
+};
```

Comment:
For consistency, prefer using named functions when declaring React components, e.g.

```
function CallHome() {
}
```

This used to matter a lot because React couldn't read the component name from an anonymous function like this. so this is the style I use (though I think this isn't the case any more). I also find that a bit easier to read than the fat arrow syntax, and at any rate let's not mix styles if we don't have to.